packages/cadl-ranch-specs/http/type/primitive/main.tsp

```diff
@@ -0,0 +1,63 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using Azure.ClientGenerator.Core;
+
+@scenarioService("/type/primitive")
+namespace Type.Primitive;
+
+@operationGroup
+@route("/string")
+interface String {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "For testing"
+  @scenario
+  @scenarioDoc("Expect to handle a string value. Mock api will return 'ok'")
```

Comment:
in your mockapi it payload seems to be `test`

Reply:
thanks. updated.